pages/_language/events/_event/index.vue

```diff
@@ -10,11 +10,15 @@
       <event-banner v-bind="internalEvent" />
     </header>
 
-    <main class="event__body">
-      <section
+    <main>
+      <div
         v-for="section in internalEvent.sections"
         :key="section.id"
         class="event-section"
+        :class="{
+          'event-section--background-black': section.backgroundColor === 'black',
```

Comment:
very specific in code and cms. make this "dark" and "darker"?

Reply:
You are right, it's not a very flexible/generic naming. Opted for explicit color names because generic terms like dark and 'darker' would be confusing for editors I think. Thoughts?

Reply:
It's okay. Let's start with this.